doc/doc-site/docs/off-chain/data-service-stack.md

```diff
@@ -1,3 +1,73 @@
 # Data service stack
 
-Stand by for more info on the Econia data service stack, which provides REST and WebSocket endpoints for Econia market and user data.
+The data service stack (DSS), is a group of programs that power the Econia off-chain products.
```

Comment:
How about "The Econia data service stack (DSS) is a collection of services that provide assorted data endpoints for integration purposes."